src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/CertificationType.vue

```diff
@@ -36,8 +36,31 @@
           </v-expansion-panel-text>
         </v-expansion-panel>
       </v-expansion-panels>
+      <h2 class="mt-10">Certified in Canada?</h2>
+      <p class="mt-3">
+        If you're certified in another province or territory in Canada, you might be able to transfer you certification to British Columbia. Check the
+        <!-- prettier-ignore -->
+        <a href="https://www2.gov.bc.ca/gov/content/education-training/early-learning/teach/training-and-professional-development/become-an-early-childhood-educator/transfer-your-certification/certification-from-province-to-province">certification levels from province to province chart</a>
+        to find out if your certification transfers.
+      </p>
```

Comment:
does this grammar make sense ? 

check the <to find out if your certification transfers.>

Reply:
@SoLetsDev  I think it's:

> Check the certification levels from province to province chart to find out if your certification transfers.